components/stepper/stories/template.js

```diff
@@ -94,3 +94,121 @@ export const Template = ({
 		</div>
 	`;
 };
+
+/* Shows all of the stepper states in one grouping. */
+export const AllDefaultVariantsGroup = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			withBorder: false,
+			containerStyles: {
+				"gap": "8px",
+			},
+			heading: "Default",
+			content: html`
+				${Template({
+					...args,
+				}, context)}
+			`
```

Comment:
```suggestion
			content: Template(args, context)
```

Reply:
There's no need to wrap the Template function in the html literal since it's returning an html template literal as well

Reply:
I was just going through the changes and noticed these. I made similar changes in steplist as well: https://github.com/adobe/spectrum-css/pull/2900/commits/7cdf8a0e1dfdf337fbf799635fd8679c16e22aeb